content/en/docs/faq.md

```diff
@@ -5,6 +5,14 @@ type: docs
 weight: 8
 ---
 
+## What registries are compatible with Notary?
+
+The following registries are compatible with Notary for artifact signing and verification:
+
+- [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry/)
+- [ORAS Distribution Registry](https://oras.land/)
```

Comment:
Can we add the exact location for download? Finding this on the web site may be cumbersome...

Reply:
@toddysm  Goot suggestion. @zr-msft Could you replace it with this URL: https://github.com/oras-project/distribution/pkgs/container/registry/64589674?tag=v1.0.0-rc.4